addon/templates/components/frost-list-content-container.hbs

```diff
@@ -1,44 +1,31 @@
 {{! Template for the frost-list-content-container component }}
+{{#vertical-collection
+  alwaysRemeasure=true
+  alwaysUseDefaultHeight=alwaysUseDefaultHeight
+  bufferSize=bufferSize
+  content=items
+  defaultHeight=defaultHeight
+  firstReached=onLoadPrevious
+  lastReached=onLoadNext
+  scrollPosition=scrollTop
+  useContentProxy=false
 
-{{#if pagination}}
-  {{#frost-scroll
-    hook=(concat hookPrefix '-scroll')
-  }}
-    {{#each items as |model index|}}
-      {{yield model index}}
-    {{else}}
-      {{yield to="inverse"}}
-    {{/each}}
-  {{/frost-scroll}}
-{{else}}
-  {{#vertical-collection
-    alwaysRemeasure=true
-    alwaysUseDefaultHeight=alwaysUseDefaultHeight
-    bufferSize=bufferSize
-    content=items
-    defaultHeight=defaultHeight
-    firstReached=onLoadPrevious
-    lastReached=onLoadNext
-    scrollPosition=scrollTop
-    useContentProxy=false
-
-    key='@identity'
-    resizeDebounce=64
-    firstVisibleChanged=null
-    lastVisibleChanged=null
-    didMountCollection=null
-    idForFirstItem=null
-    renderFromLast=false
-    renderAllInitially=false
-    shouldRender=true
-    minimumMovement=15
-    containerSelector=null
-    containerHeight=null
+  key='@identity'
+  resizeDebounce=64
```

Comment:
Not sure what this is. I simply used the same vertical collection as below... I feel like we have a lot of properties that might not be necessary in the vertical collection.